### PR TITLE
Show failure value for TCP connect on web connectivity measurement page

### PR DESCRIPTION
**Pull request by majakomel**
Closes https://github.com/ooni/explorer/issues/900

### CODE REVIEW

**Comment by majakomel** [comment]
@hellais @bassosimone How does this look? 
https://explorer-git-issue-900-ooni1.vercel.app/m/20240215101357.076638_IR_webconnectivity_c7c9fc799e116849

Should I add check mark or similar when connection succeeds so it's clearer?

**Review by bassosimone — APPROVED** [review]
🐳

File: components/measurement/nettests/WebConnectivity.js [hunk 1]
```diff
@@ -575,13 +561,7 @@ const WebConnectivityDetails = ({
...
                         <Box>
                           <Text>
-                            <FormattedMessage
-                              id='Measurement.Details.Websites.TCP.ConnectionTo'
-                              values={{
-                                destination: <strong> {connection.destination} </strong>,
-                                connectionStatus: intl.formatMessage(messages[`connection.${connection.status}`])
-                              }}
-                            />
+                            <strong>{connection.destination}</strong>{connection.failure && `: ${connection.failure}`}
```
Comment (hellais) [inline components/measurement/nettests/WebConnectivity.js:564]: I suggest we rephrase this as:
```
{destination}: {connectionStatus} (`{failureString}`)
```
Where `connectionStatus` is `failed` or `succeeded` and failure string parens only appears when it's failed and contains the value of `failure`
